app/models/user.rb

```diff
@@ -82,6 +82,7 @@ class User < ActiveRecord::Base
   has_many :muted_user_records, class_name: "MutedUser", dependent: :delete_all
   has_many :ignored_user_records, class_name: "IgnoredUser", dependent: :delete_all
   has_many :do_not_disturb_timings, dependent: :delete_all
+  has_many :sidebar_sections, dependent: :delete_all
```

Comment:
```suggestion
  has_many :sidebar_sections, dependent: :destroy_all
```

I believe we need to use `destroy_all` here to ensure that the associated records and callbacks are ran as well. Will be nice to have a spec to cover this.